src/dr_prettify.py

```diff
@@ -62,9 +62,15 @@ def pretty_ip_protocol(p):
     else:
         return p
 
+#Need configue NUM_OF_VFS to 0, otherwise the SF_BASE_INDEX may not be 2.
+SF_BASE_INDEX = 2
```

Comment:
Can we define NUM_OF_VFS and introduce it in the SF_BASE_INDEX?

Reply:
Yes, of course, will update the patch.